spread.yaml

```diff
@@ -542,7 +568,7 @@ prepare: |
         fi
     fi
 
-    if [[ "$SPREAD_SYSTEM" == debian-* ]]; then
+    if [[ "$SPREAD_SYSTEM" == debian-* ]] || ( [[ "$SPREAD_SYSTEM" == ubuntu-* ]] && [ "$SPREAD_BACKEND" == "classic-desktop" ] ); then
```

Comment:
what's the justification for this change?

Reply:
On desktop images which we don't create the eatmydata package is needed (and sometimes it is not installed). We need to manually do that because it runs before the dependency packages are installed in the machine.